### ISSUE TITLE
Rename some button sets for more natural discovery

### ISSUE THREAD

**Issue by nkmcalli** [buttonmen-dev/buttonmen#2778]
I often use the button page (https://www.buttonweavers.com/ui/buttons.html) to find button info.  Sometimes I can't find what I'm looking for because it's not where I expected it to be.  My suggestion is to rename Classic Fanatics to Fanatics Classic and to rename anything with a year at the start of the name to have the year at the end.  The advantage is that after the change all button sets that are related will appear next to each other.  Like this:

Current (related sets are all mixed up):

1999 Rare / Promo
2000 Rare / Promo
2000 ShoreCon
2002 Anime Expo
2003 Rare / Promo
2004 Origins
2005 Rare / Promo
2017 Rare / Promo
2020 Fanatics
Classic Fanatics

Suggested change (related sets are all adjacent):

Anime Expo 2002
Origins 2004
Rare / Promo 1999
Rare / Promo 2000
Rare / Promo 2003
Rare / Promo 2005
Rare / Promo 2017
ShoreCon 2000
Fanatics Classic
Fanatics 2020

**Comment by nkmcalli** [comment]
![image](https://user-images.githubusercontent.com/8868032/136877527-182652ce-297c-47eb-a50f-7a7c0326775f.png)

**Comment by blackshadowshade** [comment]
I think this is a good idea, and should be considered at the same time as #2601.

**Comment by jl8e** [comment]
Yeah, most of this goes away once the promo sets get consolidated. That just leaves fanatics and origins, I think.
